src/Markdig.Wpf/Styles.cs

```diff
@@ -86,5 +86,30 @@ public static class Styles
         /// Resource Key for the ThematicBreakStyle.
         /// </summary>
         public static ComponentResourceKey ThematicBreakStyleKey { get; } = new ComponentResourceKey(typeof(Styles), nameof(ThematicBreakStyleKey));
+
```

Comment:
At some point I should extract the ones that come with an extension into a separate class (maybe one class per extension). For the moment I prefer to have those sorted in alphabetical order.

Reply:
Sorry, I didn't notice the alphabetic ordering of style keys 😆